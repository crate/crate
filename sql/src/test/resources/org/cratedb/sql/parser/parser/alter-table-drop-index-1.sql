alter table netbeans1 drop index netbeansIDX