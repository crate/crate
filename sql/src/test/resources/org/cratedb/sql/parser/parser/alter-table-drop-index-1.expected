org.cratedb.sql.parser.parser.AlterTableNode@3b362ca8
name: netbeans1
statementType: ALTER TABLE
objectName: netbeans1
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@61ade7f6

	[0]:	
	org.cratedb.sql.parser.parser.AlterDropIndexNode@6057f993
	name: netbeansidx
	elementType: AT_DROP_INDEX
	
	existenceCheck: NO_CONDITION